tuxemon/item/conditions/shape.py

```diff
@@ -48,14 +42,18 @@ class ShapeCondition(ItemCondition[ShapeConditionParameters]):
     """
 
     name = "shape"
-    param_class = ShapeConditionParameters
+    shape1: str
+    shape2: Union[str, None]
+    shape3: Union[str, None]
```

Comment:
I think these need to be defaulted to None. Can you test?

Reply:
You're absolutely right